### ISSUE TITLE
[Fresh install] Error: ENOENT, no such file or directory

### ISSUE THREAD

**Issue by IAmJulianAcosta** [tildeio/ember-cli-fastboot#116]
Hi, I just installed fastboot for trying it, but I'm having problems with it, here is the output:

```
┌─[julian][ubuntu][±][master U:2 ?:1 ✗][~/fastboot]
└─▪ ember fastboot
version: 1.13.8
Building...Error: ENOENT, no such file or directory '/home/julian/fastboot/tmp/fast_boot_config-input_base_path-bwEVrp3Z.tmp/0/assets/assetMap.json'
    at Error (native)
    at Object.fs.openSync (fs.js:500:18)
    at Object.fs.readFileSync (fs.js:352:15)
    at FastBootConfig.readAssetManifest (/home/julian/fastboot/node_modules/ember-cli-fastboot/lib/broccoli/fastboot-config.js:69:34)
    at FastBootConfig.buildManifest (/home/julian/fastboot/node_modules/ember-cli-fastboot/lib/broccoli/fastboot-config.js:85:30)
    at FastBootConfig.build (/home/julian/fastboot/node_modules/ember-cli-fastboot/lib/broccoli/fastboot-config.js:28:8)
    at /home/julian/fastboot/node_modules/ember-cli-fastboot/node_modules/broccoli-plugin/read_compat.js:61:34
    at lib$rsvp$$internal$$tryCatch (/home/julian/fastboot/node_modules/ember-cli-fastboot/node_modules/rsvp/dist/rsvp.js:1036:16)
    at lib$rsvp$$internal$$invokeCallback (/home/julian/fastboot/node_modules/ember-cli-fastboot/node_modules/rsvp/dist/rsvp.js:1048:17)
    at lib$rsvp$$internal$$publish (/home/julian/fastboot/node_modules/ember-cli-fastboot/node_modules/rsvp/dist/rsvp.js:1019:11)
Built project successfully. Stored in "fastboot-dist".
Installing FastBoot npm dependencies
DEBUG: -------------------------------
DEBUG: Ember      : 2.3.1
DEBUG: Ember Data : 2.3.3
DEBUG: -------------------------------
Ember FastBoot running at http://[::]:3000

fastboot-dist/assets/fastboot.js:238
        Ember.$(instance.rootElement + ' .ember-view').remove();
              ^
TypeError: undefined is not a function
    at instance.didCreateRootView (fastboot-dist/assets/fastboot.js:238:15)
    at _emberRuntimeSystemObject.default.extend._setOutlets (fastboot-dist/assets/vendor.js:38691:18)
    at Queue.invoke (fastboot-dist/assets/vendor.js:11557:16)
    at Object.Queue.flush (fastboot-dist/assets/vendor.js:11621:11)
    at Object.DeferredActionQueues.flush (fastboot-dist/assets/vendor.js:11422:17)
    at Object.Backburner.end (fastboot-dist/assets/vendor.js:11800:25)
    at null._onTimeout (fastboot-dist/assets/vendor.js:12391:18)
```

Is just a blank project.

`bower.json`
```
{
  "name": "fastboot",
  "dependencies": {
    "ember": "~2.3.0",
    "ember-cli-shims": "ember-cli/ember-cli-shims#0.1.0",
    "ember-cli-test-loader": "ember-cli-test-loader#0.1.3",
    "ember-load-initializers": "ember-cli/ember-load-initializers#0.1.5",
    "ember-qunit": "0.4.9",
    "ember-qunit-notifications": "0.0.7",
    "ember-resolver": "~0.1.18",
    "jquery": "^1.11.3",
    "loader.js": "ember-cli/loader.js#3.2.1",
    "qunit": "~1.18.0"
  }
}
``` 

`package.json`
```
{
  "name": "fastboot",
  "version": "0.0.0",
  "description": "Small description for fastboot goes here",
  "private": true,
  "directories": {
    "doc": "doc",
    "test": "tests"
  },
  "scripts": {
    "build": "ember build",
    "start": "ember server",
    "test": "ember test"
  },
  "repository": "",
  "engines": {
    "node": ">= 0.10.0"
  },
  "author": "",
  "license": "MIT",
  "devDependencies": {
    "broccoli-asset-rev": "^2.1.2",
    "ember-cli": "1.13.8",
    "ember-cli-app-version": "0.5.0",
    "ember-cli-babel": "^5.1.3",
    "ember-cli-dependency-checker": "^1.0.1",
    "ember-cli-fastboot": "0.4.1",
    "ember-cli-htmlbars": "0.7.9",
    "ember-cli-htmlbars-inline-precompile": "^0.2.0",
    "ember-cli-ic-ajax": "0.2.1",
    "ember-cli-inject-live-reload": "^1.3.1",
    "ember-cli-qunit": "^1.0.0",
    "ember-cli-release": "0.2.3",
    "ember-cli-sri": "^1.0.3",
    "ember-cli-uglify": "^1.2.0",
    "ember-data": "^2.3.0",
    "ember-disable-proxy-controllers": "^1.0.0",
    "ember-export-application-global": "^1.0.3"
  }
}
```  

Am I missing something?

**Comment by tomdale** [comment]
@IAmJulianAcosta Hey, sorry you're having trouble. The first error you're seeing (`Building...Error: ENOENT, no such file or directory '/home/julian/fastboot/tmp/fast_boot_config…`) is not a problem and has been fixed in v0.4.2.

The second error you're getting is surprising. It looks similar to errors from trying to use FastBoot with older versions of Ember, but you're using 2.3.1 so it should work. Can you try upgrading to 2.4-beta.2 and see if that works?

**Comment by IAmJulianAcosta** [comment]
First error: Fixed with updating to `0.4.2`
Second error: Had to update ember cli to `1.13.15` and ember to `2.3.1` in `package.json`

Working now, closing, thanks for your valuable help!

**Comment by tomdale** [comment]
@IAmJulianAcosta Woohoo!
